src/Symfony/Component/Security/Http/Logout/LogoutHandlerInterface.php

```diff
@@ -27,5 +27,5 @@
      * to be logged out. Usually, you would unset session variables, or remove
      * cookies, etc.
      */
-    public function logout(Request $request, Response $response, TokenInterface $token);
+    public function logout(Request $request, Response $response, TokenInterface $token = null);
```

Comment:
That would fatally break any existing logout handler, not doable.
Although it's not very sexy, passing a dummy token is better as it doesn't break (I would not call it DummyToken though, we need a meaningful name), and it seems to be the best alternative we have for fixing this bug.

Reply:
Please don't close :) While it aims to achieve the same goal, it's fine to rework a PR.

Reply:
I thought we could have BC breaks between major versions?

Reply:
Not without proper upgrade path, i.e. noticing about the breaking change, that is not possible from an interface

Reply:
I presume it's too late for an upgrade path now?